pkg/controller/virtualmachinevolume/virtualmachinevolume_controller_test.go

```diff
@@ -19,20 +19,20 @@ var _ = Describe("Reconcile", func() {
 		r := createFakeReconcileVmv()
 		_, err := r.Reconcile(reconcile.Request{NamespacedName: testVolumeNamespacedName})
 
-		It("Should return error", func() {
-			Expect(err).ShouldNot(BeNil())
+		It("Should be nil", func() {
```

Comment:
'Should not return error' sounds better

Reply:
I think 'Should be nil' is more certainty. When the controller enters the 'pending' state, Reconcile 'should return nil'.